pyproject.toml

```diff
@@ -237,3 +237,11 @@ quote-style = "single"
 [tool.uv]
 # Ignore uv.lock and always upgrade the package to the latest
 upgrade-package = ["jax", "jaxlib", "orbax-checkpoint"]
+
+[tool.uv.sources]
```

Comment:
This is from https://github.com/google/flax/pull/5086 and can be removed if that PR wont land